src/frontend/src/store/index.ts

```diff
@@ -404,5 +411,53 @@ export default new Vuex.Store({
         dismissible: true,
       });
     },
+
+    ACTION_FETCH_TRANSACTION_HISTORY({ commit, getters, dispatch }) {
+      axios
+        .get("http://localhost//wallet/history?walletId="+ getters.walletId)
```

Comment:
you will need to change this to `http://localhost/wallet/history/ + getters.walletId`

Reply:
This reminded me the walletId doesn't exist anymore. You will need to use getters.umnetId